### DISCUSSION TITLE
Habit compensation

### DISCUSSION THREAD

**Discussion by amrsamii** [iSoron/uhabits#2051] [Help & FAQ]
When you miss a daily habit, do you compensate on the next day? For example if you read 10 pages every day, do you read 20 pages the next day and mark it as completed on the missed day?

**Comment by KyleSCraig** [comment]
I generally don't, but I do for laundry (which I have on an n every m days setting). But all my justifications against doing it have flaws so I think if it works for you that's great!

**Reply by amrsamii** [reply]
I think if you didn't do it, you shouldn't mark it as completed even if you compensate on the next day.

**Reply by KyleSCraig** [reply]
> I think if you didn't do it, you shouldn't mark it as completed even if you compensate on the next day.

So what's the point of this discussion? I thought you were thinking about it and asking for thoughts on it. Now it sounds like you feel strongly about your position against it. So just don't do it?

**Reply by amrsamii** [reply]
I'm still thinking. I didn't take a decision.

**Reply by KyleSCraig** [reply]
Okay in that case, some ideas: 
You're often aiming for the regularity of the habit not the volume so compensating doesn't support that goal
If you care more about volume for something like minutes reading then consider a numerical habit (I don't have any of them though anymore so I am not well informed here) 
In my laundry case it is volume I care about but also not suitable for numerical habit which is why I see it working well

**Reply by amrsamii** [reply]
I don't have numerical habits either. What do you suggest in this case?

**Reply by KyleSCraig** [reply]
Well for reading I think it's probably not a good idea, but for something like running x km per week I see it potentially making sense. So maybe you have 30km weekly goal and then just enter the kms you do every day.
